ocs_ci/framework/conf/ocp_version/ocp-4.9-config.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# Config file for nightly OCP 4.9
+RUN:
+  client_version: "4.9.0-0.nightly"
+DEPLOYMENT:
+  installer_version: "4.9.0-0.nightly"
+  terraform_version: "0.12.26"
+  # ignition_version can be found here
+  # https://docs.openshift.com/container-platform/4.7/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks
```

Comment:
Curious if this version doesn't have documentation if we can somewhere confirm the ignition_version from some other place? Maybe github and master branch could contain it somewhere.

Reply:
currently I am not sure about this version. even OCP QE is not testing 4.9 yet. I am keeping the version same as OCP 4.8 for time being.